Project/Src/Gui/TextView.cs

```diff
@@ -1062,17 +1062,41 @@ private void DrawTabMarker(Graphics g, Color color, int x, int y)
             DrawString(g, "\u00BB", tabMarkerColor.GetFont(TextEditorProperties.FontContainer), color, x, y);
         }
 
-        private int DrawEOLMarker(Graphics g, Color color, Brush backBrush, int x, int y)
+        private int DrawEOLMarker(EolMarker eolType, Graphics g, Color color, Brush backBrush, int x, int y)
```

Comment:
It is customary in .NET API to pass `Graphics` as the first argument
```suggestion
        private int DrawEolMarker(Graphics g, Color color, Brush backBrush, int x, int y, EolMarker eolMarker)
```